packages/cli/package.json

```
@@ -75,7 +76,7 @@
     "format": "prettier --write --ext '.js,.ts' **/*.ts **/*/*.ts",
     "lint:check": "eslint --ext '.js,.ts' **/*.ts",
     "lint:fix": "eslint --quiet --fix --ext '.js,.ts' **/*.ts",
-    "compile": "tsc -b tsconfig.json",
+    "compile": "tsc -b tsconfig.json && copyfiles -f src/templates/*.stub  dist/templates",
```

Maybe split this into two scripts? Like
```suggestion
    "compile": "tsc -b tsconfig.json",
    "copy-templates": "copyfiles -f src/templates/*.stub dist/templates",
```

And then invoke `npm run copy-templates` from where we're invoking `npm run compile`?

---

@NickSeagull We already use copyfiles [here](https://github.com/boostercloud/booster/blob/main/packages/framework-provider-azure-infrastructure/package.json#L54). And I think it is not possible to run two commands because they are called by lerna in every repo when you run `lerna run compile` 🥲

---

Ah nevermind, I thought this was code from the user's template!